packages/ember-simple-auth/index.js

```diff
@@ -3,37 +3,26 @@
 /* eslint-env node */
 /* eslint-disable no-var, object-shorthand, prefer-template */
 
-var writeFile = require('broccoli-file-creator');
```

Comment:
Are these deps not used anywhere else? If not, these can be removed.

Reply:
`broccoli-file-creator`, `broccoli-merge-trees`

Reply:
Good catch, solved in https://github.com/simplabs/ember-simple-auth/pull/2321/commits/5e6377b347ffcee856ded5164d178dd82e636475 thank you!